V0_Display/Documentation/Setup_and_Flashing_Guide.md

```diff
@@ -16,38 +18,89 @@
+Exit and Save
 	
-	Exit and Save
 9) Run `make clean` to clean up the make environment.
 
-10) Run `make flash FLASH_DEVICE=xxxx:yyyy` (using xxxx:yyyy from above)
+10) Run `make flash FLASH_DEVICE=xxxx:yyyy` (using xxxx:yyyy from above) [0483:df11]
 
 11) After completion `ls /dev/serial/by-id/*` should return a device begining with `/dev/serial/by-id/usb-Klipper_stm32f042x6...`
-
-Your board should now be usable with Klipper. Use the example config file to get started
-Best option is to copy the config file into the same directory as `printer.cfg` then add `[include DisplayEncoder.cfg]` to the end of your `printer.cfg` to include the file.
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+Example of `ls /dev/serial/by-id/*` note the MCU will also be displayed if attached
+
+	/dev/serial/by-id/usb-Klipper_stm32f042x6_050000000D43535035373520-if00
+	/dev/serial/by-id/usb-Klipper_stm32f103xe_30FFD3054255353817721057-if00
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+
+12) Open Mainsail/Fluidd in browser and edit `printer.cfg`, add the following to a location of your liking
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+	# Required to enable V0 display
+	[include V0_Display.cfg]
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+Save and close `printer.cfg`
+
+13) Create a new file in Mainsail/Fluidd callled `V0_Display.cfg`
+
+14) Edit `V0_Display.cfg` and copy the following, ensuring to use the serial address from step 11
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+	[mcu displayEncoder]
+	serial: /dev/serial/by-id/usb-Klipper_stm32f042x6_050000000D43535035373520-if00
+	restart_method: command
+
+	[display]
+	lcd_type: sh1106
+	i2c_mcu: displayEncoder
+	i2c_bus: i2c1a
+	encoder_pins: ^!displayEncoder:PA4, ^displayEncoder:PA3
+	click_pin: ^!displayEncoder:PA1
+	kill_pin: ^!displayEncoder:PA5
+	vcomh: 17
+	#   Set the Vcomh value on SSD1306/SH1106 displays. This value is
+	#   associated with a "smearing" effect on some OLED displays. The
+	#   value may range from 0 to 63. Default is 0.
+	#   Adjust this value if you get some vertical stripes on your display. (31 seems to be a good value)
+	
+	#   RGB Values should be set to match your printer colour scheme
+	[neopixel displayStatus]
+	pin: displayEncoder:PA0
+	chain_count: 1
+	color_order: GRB
+	initial_RED: 0.01
+	initial_GREEN: 0
+	initial_BLUE: 0.01
```

Comment:
I would remove this and direct the user to either UPLOAD or COPY the contents of the config file in the GitHub so if i make any changes i dont have to change it in two places.

13) Download the (link to CFG) file and upload that to the same folder as your printer.cfg.